pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -285,6 +283,16 @@ func (r *ReconcileHyperConverged) Reconcile(ctx context.Context, request reconci
 	return res, err
 }
 
+func isTriggeredByHyperConverged(request reconcile.Request) (bool, error) {
+	placeholder, err := getSecondaryCRPlaceholder()
+	if err != nil {
+		return false, err
+	}
+
+	isSecondary := request.NamespacedName == placeholder
+	return !isSecondary, nil
```

Comment:
Can we return the actual boolean value instead of its opposite value? I find it a bit confusing. Something like:

```golang
        isHyperConvergedChange := request.NamespacedName != placeholder
	return isHyperConvergedChange, nil
```

Reply:
We could, it's just a trade of where we put the negation. I thought it's clearer as-is, but don't mind changing.

Reply:
Changed.